java/src/jmri/jmrit/symbolicprog/SplitVariableValue.java

```diff
@@ -667,7 +669,8 @@ public void readAll() {
         }
         _progState = READING_FIRST;
         retry = 0;
-        log.debug("Variable={}; invoke CV read", _name);
+        log.info("Variable={}; Start CV read", _name);
```

Comment:
@TCSDCC-Jordan I've looked at the changes to this file. If you simply replace all "log.info" with "log.debug", this file should be good to go. It's too noisy to commit as-is.

Reply:
Since SplitEnumVariableValue no longer extends SplitVariableValue, there's no need to include this file in #10569. You could just revert it.

 I've got the SplitVariableValue changes in a local fork so I can review my changes and commit them as a separate PR if desirable.

Reply:
@dheap If you could take care of reverting this file in a separate PR that would be much appreciated.

Reply:
@TCSDCC-Jordan 
We don't want to merge PRs with known errors. If you want, I could help you with reverting the changes in this file in this PR. But you would then need to pull the changes I make in this PR to your local repository.

Reply:
@danielb987, two possible suggested ways forward with this file:

1. Just change all log.info to log.debug.
2. Revert all changes.

Either way, I can easily compare with my fork and create a separate PR later if other changes are desirable.

Reply:
@dheap @danielb987 I just did 1 from Dave's options above. Thanks Dave, I wasn't fully understanding.

Reply:
@TCSDCC-Jordan 
Thanks.

Reply:
@TCSDCC-Jordan @danielb987 This file now matches the copy in my local fork. Thanks.